### ISSUE TITLE
How do you customize the email message

### ISSUE THREAD

**Issue by NicolasDhaene** [bee-keeper/django-invitations#138]
I would like to add element of context like the inviter's name.
Is this possible?

**Comment by dspec12** [comment]
Override `email_invite_message.txt` in  `django-invitations/invitations/templates/invitations/email/`. 

You can do this by  creating the file in the same directory path in your project.

**Comment by pcraig3** [comment]
Yeah, if this isn't clear, you can create an `.html` file in your project at `{projectroot}/{app}/templates/invitations/email/email_invite_message.html` and it will override [the default template].

**Comment by jonathanccardoso** [comment]
> Yeah, if this isn't clear, you can create an `.html` file in your project at `{projectroot}/{app}/templates/invitations/email/email_invite_message.html` and it will override [the default template](https://github.com/bee-keeper/django-invitations/blob/master/invitations/templates/invitations/email/email_invite_message.txt).

This worked for me.

**Comment by avenyproduction** [comment]
> Yeah, if this isn't clear, you can create an `.html` file in your project at `{projectroot}/{app}/templates/invitations/email/email_invite_message.html` and it will override [the default template](https://github.com/bee-keeper/django-invitations/blob/master/invitations/templates/invitations/email/email_invite_message.txt).

What about email subject? Should it be in the same directory? (I have tried with email_invite_subject.txt but no luck. email_invite_message.txt didn't work for me, but email_invite_message.html did )

> Yeah, if this isn't clear, you can create an `.html` file in your project at `{projectroot}/{app}/templates/invitations/email/email_invite_message.html` and it will override [the default template](https://github.com/bee-keeper/django-invitations/blob/master/invitations/templates/invitations/email/email_invite_message.txt).

What about email subject? Should it be in the same directory? (I have tried with email_invite_subject.txt but no luck. email_invite_message.txt didn't work for me, but email_invite_message.html did )

**Comment by matthew-a-dunlap** [comment]
> > Yeah, if this isn't clear, you can create an `.html` file in your project at `{projectroot}/{app}/templates/invitations/email/email_invite_message.html` and it will override [the default template](https://github.com/bee-keeper/django-invitations/blob/master/invitations/templates/invitations/email/email_invite_message.txt).
> 
> What about email subject? Should it be in the same directory? (I have tried with email_invite_subject.txt in the same directory, but no luck)

It looks like the code isn't as flexible for the templates other than `email_invite_message`. [This may be a good place to start looking](https://github.com/bee-keeper/django-invitations/blob/9069002f1a0572ae37ffec21ea72f66345a8276f/invitations/adapters.py#L34).
